### ISSUE TITLE
Apiary (Aragon DAO Explorer)

### ISSUE THREAD

**Issue by lkngtn** [1Hive/ideas#32]
Since the bonding curve app aspect of **Apiary** was renamed **Aragon Fundraising** and is being developed and maintained by Aragon Black. I propose shifting the focus of Apiary to providing a portal to help browse and participate specifically in open source projects using Aragon Fundraising.

**Comment by yeqbfgxjiq** [comment]
That sounds awesome. We could probably even reuse whatever UI we create for the HCL Registry, but then use it for Fundraising Apps (list of projects, search, etc...). It might be a bit weird/limited to _just_ display projects that use that specific Aragon app. What about a portal that talked about the various funding mechanisms that open source projects are using, and the Aragon Fundraising app could be one of those. Then it would be a portal for open source projects could come learn about the various funding models available, see projects that are using them, and then see templates for things they could use themselves.

**Comment by lkngtn** [comment]
Generally I’m a bit wary of going down a more general path, sticking to just aragon orgs, or just Aragon orgs using fundraising, or even just Aragon orgs using a specific template, would help keep the scope narrow and focused.

We can better provide consistency for users, and can better integrate features because the tech “stack” will be consistent.

**Comment by yeqbfgxjiq** [comment]
I agree that it will be easier to integrate features if people coming to the site/resource are already using Aragon apps/DAOs. That being said, the HCL isn't an Aragon app, yet it's another viable tool that can help improve sustainability. There are and are going to be lots of tools that can help open source committees thrive. I'd love to see us get involved in using whatever tools/resources we need to help open source projects thrive. Limiting our scope to _just_ Aragon related things makes 1Hive more of an Aragon project than an open source sustainability project. That's not as exciting to me, and frankly that's not the mission of 1Hive.

**Comment by lkngtn** [comment]
> Limiting our scope to just Aragon related things makes 1Hive more of an Aragon project than an open source sustainability project

The idea here is for Apiary to be essentially a DAO explorer, to help people find and participate in DAOs. The intention behind limiting the scope to Aragon here is because we can provide a better explorer if the underlying DAOs all use the same architecture. You wouldn't try and create a block explorer that works for both bitcoin and ethereum? Its sort of the same thing here. 

When it comes to DAO related projects I think it makes sense for us to be opinionated about the DAO framework we are using. That doesn't mean that the HCL license has a dependency on Aragon, or that 1Hive will only ever work on Aragon related initiatives... 

At the end of the day the mission is to empower open source projects, and personally I think DAOs are a big part of that.

**Comment by yeqbfgxjiq** [comment]
Ah. So you're thinking an automated Aragon block explorer that displays stats on DAOs that use the Fundraising app. I was thinking of a platform/community that crowdsources info on open source projects that are using sustainable fundraising models (perhaps even with discussion boards and a wiki with recommended resources).

**Comment by lkngtn** [comment]
After speaking with @onbjerg today, we determined that his project daolist is an excellent base for a Aragon dao explorer. 

Coincidently he had just decided to stop maintaining it, so I figured we could adopt it. We have open sourced the code for daolist, and migrated it to https://github.com/1hive/apiary (I moved the previous apiary repo to https://github.com/1hive/apiary-paper and may update/merge them later)  

Also apiary.1hive.org now hosts the project and can serve as a replacement for daolist now that it has been taken down. Will add issues in the repo for future improvements, and close this idea.
